### PR TITLE
Meta: anchor HR-TIME and RESOURCE-TIMING terms

### PR DESCRIPTION
**Pull request by annevk**
ReSpec-authored specifications are best anchored as otherwise outgoing links go all over the map.

***
<a href="https://whatpr.org/fetch/1224.html" title="Last updated on May 25, 2021, 6:42 PM UTC (8080de9)">Preview</a> | <a href="https://whatpr.org/fetch/1224/208e271...8080de9.html" title="Last updated on May 25, 2021, 6:42 PM UTC (8080de9)">Diff</a>

### CODE REVIEW

**Comment by annevk** [comment]
@deniak it seems I somehow still end up with hr-time-2 (see the Preview link). But also resource-timing-1, so maybe the problem is with Bikeshed @tabatkins?

Also somehow DOMHighResTimeStamp is not picked up from the anchors block (but no error is output either).

**Comment by deniak** [comment]
After a closer look, it appears that the alias is still needed as specref still relies on tr.rdf to fetch the data instead of the W3C API. I've submitted a [PR to restore the alias](https://github.com/tobie/specref/pull/654). Sorry for asking you to remove it earlier.
Now, for `resource-timing`, it seems the [`w3c.json`](https://github.com/tobie/specref/blob/main/refs/w3c.json) was last updated on April 22nd so it didn't pick up the new aliases. @tobie, is it supposed to be updated [every hour as mentioned in the README](https://github.com/tobie/specref#hourly-auto-updating)?

**Comment by tobie** [comment]
> After a closer look, it appears that the alias is still needed as specref still relies on tr.rdf to fetch the data instead of the W3C API. I've submitted a [PR to restore the alias](https://github.com/tobie/specref/pull/654). Sorry for asking you to remove it earlier.

Merged.

> Now, for `resource-timing`, it seems the [`w3c.json`](https://github.com/tobie/specref/blob/main/refs/w3c.json) was last updated on April 22nd so it didn't pick up the new aliases. @tobie, is it supposed to be updated [every hour as mentioned in the README](https://github.com/tobie/specref#hourly-auto-updating)?

The changes that were made to those aliases broke a number of tests which now fail and so rejects new updates.

**Comment by deniak** [comment]
> The changes that were made to those aliases broke a number of tests which now fail and so rejects new updates.

Ah, thanks for checking. I've submitted a [PR to fix the issue](https://github.com/tobie/specref/pull/656).

**Comment by deniak** [comment]
@annevk specref should return the right versions

**Comment by annevk** [comment]
Thanks @deniak! It seems to be mostly in order, except for DOMHighResTimeStamp, which goes to hr-time-2 somehow. @tabatkins, can you help with that? I suspect I'm holding Bikeshed wrong.

File: fetch.bs [hunk 1]
```diff
@@ -11,15 +11,28 @@ Translate IDs: typedefdef-bodyinit bodyinit,dictdef-requestinit requestinit,type
...
+    url:section-4.2;text:parsing structured fields
+
+url:https://w3c.github.io/resource-timing/#dfn-mark-resource-timing;text:mark resource timing;type:dfn;spec:resource-timing
+
+urlPrefix:https://w3c.github.io/hr-time/#;spec:hr-time
+    type:dfn
+        url:dfn-coarsen-time;text:coarsen time
+        url:dfn-coarsened-shared-current-time;text:coarsened shared current time
+        url:dfn-unsafe-shared-current-time;text:unsafe shared current time
+    type:idl;url:dom-domhighrestimestamp;text:DOMHighResTimeStamp
```
Comment (tabatkins) [inline fetch.bs:35]: "idl" is a linking supertype, not a valid type for the dfns themselves. Bikeshed not complaining about that is my fault; filed <https://github.com/tabatkins/bikeshed/issues/2068>

```suggestion
    type:typedef;url:dom-domhighrestimestamp;text:DOMHighResTimeStamp
```

**Pull request merged**